server/metadata.go

```diff
@@ -1,6 +1,22 @@
 package server
 
+import "fmt"
+
 const role_label string = "com.planetary-quantum.meta.role"
+const group_label string = "com.planetary-quantum.meta.customer_group"
+
+func GetGroup(server AnsibleServer) (string, error) {
```

Comment:
This is where I extract the name from the instance metadata.

Reply:
Not really sure if I use the constants correctly. Maybe this should be something else. Basically looking to declare a type that I can consume in other apps as well. But no idea, this also works.

Reply:
yeah I think this is okay